tools/bazel.rc

```diff
@@ -1,5 +1,8 @@
 # TODO(#2897): move to %workspace%/.bazelrc
 
+# Remote Build Execution requires a strong hash function, such as SHA256.
+startup --host_jvm_args=-Dbazel.DigestFunction=SHA256
```

Comment:
Turns out you can't put a startup option under a config.

Reply:
Oh. I guess I should have realized that, since presumably the configs aren't read till after startup. ☹️